projects/common/src/time/time-duration.ts

```diff
@@ -26,6 +29,70 @@ export class TimeDuration {
     return `PT${this.toMillis() / 1000}S`;
   }
 
+  public toIso8601DurationStringMilitaryTime(): string {
+    const timeObject = this.toTimeObject();
```

Comment:
Likewise, avoid stateful calls. I'm not sure why we need both a date object and a time object, they both accomplish the same thing.